rs/interfaces/src/execution_environment.rs

```diff
@@ -175,6 +175,7 @@ pub enum SystemApiCallId {
     IsController,
     /// Tracker for `ic0.mint_cycles()`
     MintCycles,
+    /// // TODO?
```

Comment:
Is this marker for the "high level" mint cycles api or do we use it to track actual syscalls?